web-ng/src/app/services/router/router.service.ts

```diff
@@ -184,26 +168,31 @@ export class NavigationService {
    * Navigate to the current URL, updating the observation id in the URL
    * fragment.
    */
-  setObservationId(id: string | null) {
-    this.setFragmentParam(
-      Map([[NavigationService.OBSERVATION_ID_FRAGMENT_PARAM, id]])
-    );
+  editObservation(featureId: string | null, observationId: string) {
```

Comment:
When would `editObservation()` be called without `featureId`? Maybe this should be required?